client/my-sites/people/invite-people/index.jsx

```diff
@@ -46,18 +56,41 @@ export default React.createClass( {
 			role: 'follower',
 			message: '',
 			response: false,
-			sendingInvites: false
+			sendingInvites: false,
+			getTokenStatus: () => {}
 		} );
 	},
 
 	onTokensChange( tokens ) {
 		this.setState( { usernamesOrEmails: tokens } );
+		const { siteId, role } = this.state;
+		createInviteValidation( siteId, tokens, role );
 	},
 
 	onMessageChange( event ) {
 		this.setState( { message: event.target.value } );
 	},
 
+	refreshValidation() {
+		const errors = InvitesCreateValidationStore.getErrors( this.state.siteId ) || [];
+		let success = InvitesCreateValidationStore.getSuccess( this.state.siteId ) || [];
+		if ( ! success.indexOf ) {
+			success = Object.keys( success ).map( key => success[ key ] );
+		}
+		this.setState( {
+			getTokenStatus: ( value ) => {
```

Comment:
Should be about the same, besides, the function behaviour really changes, I like that we are creating a new instance.